### DISCUSSION TITLE
Student.java dosyasinin repoya eklenmeme durumu

### DISCUSSION THREAD

**Discussion by ozitoman** [gusanmaz/BMB212-Algorithms-2223#17] [Q&A] [answered]
Ben ilk beş ödevi yaptım ancak ilk ödevde sizin veriğiniz kodu kendi kodlarıma eklemediğimi fark ettim. Eklemek zorunda mıyım yoksa eklemesem de kabul eder misiniz?

**Comment by gusanmaz** [comment, marked as answer]
Evet, eklemeniz gerekiyor. Reponuzdaki .java uzantili dosyalarin hepsini kullanarak kodunuzu derleyebilmem gerekiyor. 

Herkesin kendi repsouna ekleyecegi `Student.java` dosyasinin kodlari ayni olacak. `Student.java` dosyasinin kodlari size odev aciklamalarinda hazir olarak veriliyor ama yine de bu dosyayi reponuza eklemeniz gerekiyor.
